.gitleaks.toml

```diff
@@ -10,7 +25,17 @@
 [[ rules ]]
     id = "generic-api-key"
     [ rules.allowlist ]
-        commits = [
-            "474554504e7e33cef2a71774f825d5b3947ff797",
-            
+
+        paths = [
+            # Allowlists a false positive detection at  
+            # https://github.com/apollographql/apollo-ios/blob/474554504e7e33cef2a71774f825d5b3947ff797/Tests/ApolloCodegenTests/TestHelpers/ASTMatchers.swift#L72
+            # This was previously allowlisted via commit hash, but updating that rule
+            # To support allowlisting false positive detections in the files below as well.
+            '''Tests/ApolloCodegenTests/TestHelpers/ASTMatchers.swift''',
```

Comment:
@BobaFetters - this file location will need to change with the move to apollo-ios-dev right?

@peakematt - you may not know about the upcoming change to our repo structure which is going to affect these changes. You can read about the change [here](https://github.com/apollographql/apollo-ios/issues/3240). It's probably still worth merging in these changes and then adjusting the secret detection once we do the merge of that change.

Reply:
Thank you! That's really good to know! 

Yeah I'll get these merged in and then can make adjustments to the new repos if/when needed.

Reply:
Yea the `Tests` folder in its entirety is going to exist in the new `apollo-ios-dev` repo so will probably just need to move this there.